src/app/auth/change_password.tsx

```diff
@@ -0,0 +1,55 @@
+import { Redirect, router } from 'expo-router';
+import React, { useState } from 'react';
+import { Alert, View } from 'react-native';
+import { Button, Input } from 'react-native-elements';
+
+import globalStyles from '../../styles/globalStyles';
+import { useSession } from '../../utils/AuthContext';
+
+function VerificationScreen() {
+  const { updateUser, user } = useSession();
+  const [password, setPassword] = useState('');
+  const [loading, setLoading] = useState(false);
+
+  const changePassword = async () => {
+    setLoading(true);
+    const { error } = await updateUser({ password });
+
+    if (error) {
+      console.error(error);
+      Alert.alert('Updating password failed');
+    } else {
+      router.replace('/home');
+    }
+
+    setLoading(false);
+  };
+
+  if (!user) return <Redirect href="/auth/login" />;
```

Comment:
I'm getting a bug with this redirect after going through the forgot password flow:
- First, run through the forgot password flow for an existing account
- Once logged in, go to settings and click the change password button
- Instead of being redirected to the change password screen, I get redirected to the login screen

This is happening because the session is not getting reset when you call updateUser. If we wanted to keep the flow as is, we would want to reset the session in updateUser; however, we're going to follow the convention that once you reset a password, you're logged out and have to log back in with the new password. Please edit the flows for both forgot password and reset password such that once a password is successfully reset, the user is immediately logged out and session set to null.

Reply:
Does this not matter anymore because we are removing reset password? @akshaynthakur

Reply:
The logic should still apply because we're keeping forgot password and want to make sure the session is being properly reset after a user changes their password. The fix is pretty easy because you'll just log out a user after they choose a new password in the forgot password flow and redirect them to the login screen (which once done successfully should set the session and direct them home)